src/core/public/chrome/nav_controls/nav_controls_service.ts

```diff
@@ -105,6 +110,11 @@ export class NavControlsService {
           new Set([...navControlsExpandedCenter$.value.values(), navControl])
         ),
 
+      registerLeftBottom: (navControl: ChromeNavControl) =>
+        navControlsLeftBottom$.next(
+          new Set([...navControlsLeftBottom$.value.values(), navControl])
```

Comment:
does order matters?

Reply:
It matters, the items will be sorted by order when calling `getLeftBottom$()`